packages/react-components/react-utilities/src/compose/constants.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @internal
+ * internal symbol used to keep defaultProps.children available
+ */
+export const slotInternalChildrenSymbol = Symbol('fui.slotInternalChildren');
+/**
+ * @internal
+ * internal symbol used to keep slot.children available
+ */
+export const slotExternalChildrenSymbol = Symbol('fui.slotExternalChildren');
```

Comment:
```suggestion
export const SLOT_EXTERNAL_CHILDREN_SYMBOL = Symbol('fui.slotExternalChildren');
```

nit: is it a constant?